huaweicloud/provider.go

```diff
@@ -2228,6 +2228,7 @@ func Provider() *schema.Provider {
 
 			"huaweicloud_cdm_cluster_v1": cdm.ResourceCdmCluster(),
 			"huaweicloud_css_cluster_v1": css.ResourceCssCluster(),
+			"huaweicloud_css_restore":    css.ResourceCssRestore(),
```

Comment:
please move it to line 1480-1498

Reply:
may be to 1464? after huaweicloud_css_snapshot

Reply:
ok